MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -181,6 +184,23 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         let customPermission = self.miniAppKeyStore.getCustomPermissions(forMiniApp: self.miniAppId).filter { $0.permissionName == customPermissionType }
         return customPermission[0].isPermissionGranted.boolValue
     }
+
+    func setScreenOrientation(requestParam: RequestParameters?, callbackId: String) {
```

Comment:
On Android, we have added a setting for `MiniAppMessageBridge.allowScreenOrientation`. So if the Host App doesn't enable screen orientation changes, then the SDK won't do anything when the mini app calls `setScreenOrientation`.

Does this make sense to add on iOS? If the Host App doesn't override `UIViewController. supportedInterfaceOrientations` will the SDK be able to change anything about the orientation?

Reply:
I don't think we need this in iOS. We have implemented it differently in iOS due to the delegation pattern for orientation.

`func setScreenOrientation(requestParam: RequestParameters?, callbackId: String)` will only lock/set the orientation for the mini-app/video and the Host app must use `self.miniAppDisplayDelegate?.getSupportedOrientation()` in DisplayController or any viewcontroller which they are displaying Mini-app view.
And `self.miniAppDisplayDelegate?.getSupportedOrientation()` i.e MiniAppDisplayProtocol will retrieve the orientation supported for that mini-app.